geniza/corpus/templates/admin/corpus/document/change_form.html

```diff
@@ -0,0 +1,15 @@
+{% extends "admin/change_form.html" %}
+{% block after_field_sets %}
+<fieldset class="module aligned">
+    <div class="form-row">
+        <div class="fieldBox">
+            <label>First input:</label>
+            {% include "admin/corpus/document/rev_log_entry.html" with log_entry=first_input %}
```

Comment:
couldn't you just pass `instance.log_entries.first` here and not need the view method?